pages/acceptInvite.page.tsx

```diff
@@ -0,0 +1,99 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import React, { ReactElement, useEffect } from 'react';
+import { getSession } from 'next-auth/react';
+import { useSnackbar } from 'notistack';
+import { useTranslation } from 'react-i18next';
+import { SetupPage } from 'src/components/Setup/SetupPage';
+import useGetAppSettings from 'src/hooks/useGetAppSettings';
+import { loadSession } from './api/utils/pagePropsHelpers';
+
+const AcceptInvitePage = (): ReactElement => {
+  const { t } = useTranslation();
+  const { appName } = useGetAppSettings();
+  const { enqueueSnackbar } = useSnackbar();
+  const router = useRouter();
+
+  useEffect(() => {
+    if (!router.isReady) {
+      return;
+    }
+    const { accountListId, inviteCode, accountInviteId, orgInviteId, orgId } =
+      router.query;
+    const acceptInvite = async (id, code, url) => {
+      const inviteType = url.includes('organizations')
+        ? 'organization_invites'
+        : 'account_list_invites';
+
+      try {
+        const session = await getSession();
```

Comment:
You can do useRequiredSession to get the token. Can you move the fetch to a helper method outside of the component?

Reply:
For moving the fetch, are you thinking I just pass all the props to it? Like `apiToken`, `code`, `url`, `id`

Reply:
I think so!